app/javascript/spree/dashboard/index.js

```diff
@@ -1,53 +1,46 @@
-//
-// Initialize Dashboard
-class Dashboard {
-  constructor() {
-    console.log("Spree Dashboard Initialized")
-  }
-}
+/* eslint-disable no-undef */
 
-//
-// Import JavaScript packages that are required globally.
+/////////////////////////
+// Node Module Imports //
+/////////////////////////
+import "@hotwired/turbo-rails"
```

Comment:
@MatthewKennedy if we import it here it will cause a double loading error in the parent application

Reply:
Hi @damianlegawiec,

This is true if the dev imports a second custom javascript file into the spree-dashboard views, with the second custom javascript file also using `import "@hotwired/turbo-rails"`. 

This is why the bulk of the tests fail in CI, because although "@hotwired/turbo-rails" is loaded in `application.js`, `application.js` is never used in the dashboard views and so its never present in the CI and the tests implode.

I have tried several  fix options, but all my previous findings were that `require` is `undefined` and causes an error in one of the build tools. I tested this on a new rails 7 app using the `importmaps`, `esbuild` and `rollup`, some work some, while others throw an error `require` is `undefined` and crash the javascript.

After all the testing I have come to the conclusion, that ideally anyone building anything that extends on top of spree-dashboard.js and looking to create some custom javascript should create `custom-spree-dashboard.js` load that in the spree dashboard views and intentionally not `import "@hotwired/turbo-rails"` a second time in that file.